light_node/src/main.rs

```diff
@@ -370,21 +349,67 @@ fn block_on_actors(
     )
     .expect("Failed to create RPC server");
 
-    // TODO: TE-386 - controlled startup
-    std::thread::sleep(std::time::Duration::from_secs(2));
+    if let Some(mut blocks) = blocks_replay.take() {
+        let chain_id = init_storage_data.chain_id.clone();
+        let starting_block = blocks.remove(0);
+        let batch = ApplyBlockBatch::batch(starting_block, blocks);
+        let requester = DataRequesterRef::new(DataRequester::new(
+            BlockMetaStorage::new(&persistent_storage),
+            OperationsMetaStorage::new(&persistent_storage),
+            block_applier,
+        ));
+
+        requester.call_schedule_apply_block(Arc::new(chain_id), batch, None);
+    } else {
+        if env.p2p.disable_mempool {
```

Comment:
@sebastiencs 
why do we need to turn-off these actors?
for example, we could tune with reaply also Monitor/MempoolPrevalidator and so ...

Reply:
@bkontur Ah ok, I thought they were not useful.
The actor `PeerManager` can be turned off ?

Reply:
@sebastiencs 
yes, PeerManager can be turned off, I guess it has to be :)